src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
```

Comment:
redundant. Just add display: inline-block (or block) to an image to remove extra line spacing
<img width="169" alt="Screenshot 2023-07-17 at 21 36 13" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/7b7a5c85-9f0e-4bd9-b5e8-8ec1de41ca8a">
```suggestion
.logo__image {
  display: block;
  width: 40px;
  height: 40px;
}
```